examples/colab/tf2_hub_single_image_super_resolution_ESRGAN.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<p>Please split the functionality of <code>plot_image</code> into <code>maybe_clip_image</code>, <code>plot_image</code> and <code>save_image</code> . There is no reason why plotting function should be doing saving.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/hub/pull/352/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Okay! :)</p>